aws/main.tf

```diff
@@ -87,17 +96,16 @@ terraform {
   }
 }
 
-variable "region" {
+variable "subnet_id" {
+  default = ""
   type    = string
-  default = "us-west-2"
 }
 
-provider "aws" {
-  region = var.region
+variable "aws_region" {
```

Comment:
Do we need to change the naming of the variable here? This might break users and also I think `region` it's fine given that we are running the terraform script in aws.
 
 In the past, there was some work done trying to standardize TF variable naming across different cloud providers and I think that's why we had only `region` here.

Reply:
I'm fine with it either way, but I worry it could be confusing with the module referring to aws_region while this refers to region.

Reply:
I'll leave it up to you, I just don't like making breaking changes as you may have noticed :rofl: 

I'll approve

Reply:
My viewpoint is that breaking stuff en route to consistency is worth it, especially when 

* these are examples rather than something people depend on
* rolling back or making a change to accommodate it is simple

Both are true in this case.

Reply:
So long as we have proper version bumping.